app/src/main/resources/application.properties

```diff
@@ -5,6 +5,7 @@ quarkus.index-dependency.jaxrs.group-id=org.jboss.spec.javax.ws.rs
 quarkus.index-dependency.jaxrs.artifact-id=jboss-jaxrs-api_2.1_spec
 
 # Name and description
+registry.id=apicurio-registry
```

Comment:
Quarkus automatically translates ENV vars for every property.  So if you have a property named `registry.id` then you can use an ENV var of `REGISTRY_ID` without doing anything extra.

So typically I only add the `${APICURIO_REGISTRY:apicurio-registry}` if I want to support an ENV var with a different name than what Quarkus would allow by default.  This has been learnt over time though, so there are probably inconsistencies.